lms/djangoapps/course_home_api/outline/v1/tests/test_views.py

```diff
@@ -79,6 +79,8 @@ def test_get_authenticated_enrolled_user(self, enrollment_mode):
     @override_experiment_waffle_flag(COURSE_HOME_MICROFRONTEND, active=True)
     @override_waffle_flag(COURSE_HOME_MICROFRONTEND_OUTLINE_TAB, active=True)
     def test_get_authenticated_user_not_enrolled(self):
+        CourseEnrollment.enroll(self.user, self.course.id)
+        CourseEnrollment.unenroll(self.user, self.course.id)
```

Comment:
Maybe also adding a small comment above these as well where you state this is in order to create the enrollment object and then set is_active to False.